objects/src/errors.rs

```diff
@@ -145,7 +145,7 @@ pub enum AssetError {
         other_issuer: AccountId,
     },
     #[error("faucet account id in asset is invalid")]
-    InvalidFaucetAccountId(#[source] Box<dyn Error + Send>),
+    InvalidFaucetAccountId(#[source] Box<dyn Error + Send + Sync>),
```

Comment:
`'static` is also required for `anyhow::Error::new` (https://docs.rs/anyhow/latest/anyhow/struct.Error.html#method.new), so we should also add that.